integration/tests/conftest.py

```diff
@@ -40,18 +40,23 @@ def pytest_collection_modifyitems(config, items):
 
     if network_name != "night-stand":
         deselected_marks.append("slow")
+    
+    if network_name == "mainnet":
+        deselected_marks.append("only_stands")
+        deselected_marks.append("only_devnet")
```

Comment:
Is it needed? I thought if you run tests with only mark "mainnet"  , all other marks automatically deselect

Reply:
Yes, you're right. Everything works without this marks. I deleted it